Packs/Accessdata/Integrations/AccessdataV2/AccessdataV2.py

```diff
@@ -0,0 +1,304 @@
+}
+
+@wrap_demisto_command("accessdata-api-create-filter")
+def _create_filter(client, column, comparator, value):
+    comp = _comparator_mapping.get(comparator, comparator)
+    # get the relevant attribute
+    attr = client.attributes.first_matching_attribute("attributeUniqueName", column)
+    # if not exists, raise error
+    if not attr:
+        demisto.error(f"Cannot find attribute of name ({column}).")
+    # if datatype is int, convert value
+    ##if attr["dataType"] & AttributeType.INT_ALL == attr["dataType"]:
+    ##    value = int(value)
+    # try get comparison method
+    has_comp = hasattr(attr, comp)
+    if not has_comp:
+        demisto.error(f"Cannot find compare method of method name ({comp}).")
+    # compare and return
+    filter_json = getattr(attr, comp)(value)
+    return CommandResults(
+        outputs_prefix="Accessdata",
+        outputs={
+            "Filter": dumps(filter_json)
+        }
+    )
+
+@wrap_demisto_command("accessdata-api-combine-filter-and")
+def _and_filter(client, filter_json1, filter_json2):
+    # try find json in the filters
+    try:
+        filter_json1 = loads(filter_json1)
+        filter_json2 = loads(filter_json2)
+    except:
+        demisto.error("Both filters must be JSON content.")
+        return CommandResults(
+            outputs_prefix="Accessdata",
+            outputs={}
+        )
+
+    filter_json = and_(filter_json1, filter_json2)
+    return CommandResults(
+        outputs_prefix="Accessdata",
+        outputs={
+            "Filter": dumps(filter_json)
+        }
+    )
+
+@wrap_demisto_command("accessdata-api-combine-filter-or")
+def _or_filter(client, filter_json1, filter_json2):
+    # try find json in the filters
+    try:
+        filter_json1 = loads(filter_json1)
+        filter_json2 = loads(filter_json2)
+    except:
+        demisto.error("Both filters must be JSON content.")
+        return CommandResults(
+            outputs_prefix="Accessdata",
+            outputs={}
+        )
+
+    filter_json = or_(filter_json1, filter_json2)
+    return CommandResults(
+        outputs_prefix="Accessdata",
+        outputs={
+            "Filter": dumps(filter_json)
+        }
+    )
+
+""" Entry Point """
+
+# gather parameters
+params = demisto.params()
+
+# generate client arguments
+protocol = params.get("PROTOCOL", "http")
+port = params.get("PORT", "4443")
+address = params.get("SERVER", "localhost")
+url = f"{protocol}://{address}:{port}/"
+apikey = params.get("APIKEY", "")
+# check if using ssl
+is_secure = protocol[-1] == 's'
+
+# build client
+client = Client(url, apikey, validate=not is_secure)
+# if using ssl, gather certs and apply
+if is_secure:
+    public_certificate = params.get("PUBLIC_CERT", None)
+    client.session.cert = public_certificate
+
+# call function with supplied args
+command = demisto.command()
+func = _run_functions[command]
+args = demisto.args()
+
+# return value from called function
+return_values = func(client, **args)
```

Comment:
Surround with try except, and except errors like [here](https://github.com/demisto/content/blob/master/Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py#L1452).